suite-common/graph/src/hooks.ts

```diff
@@ -46,7 +47,7 @@ export const useGraphForAccounts = ({
 
             try {
                 const points = await getMultipleAccountBalanceHistoryWithFiat({
-                    accounts,
+                    accounts: accounts.filter(account => !isTestnet(account.coin)),
```

Comment:
I think this should be better handled in place where data are passed for this hook. For example have selector that selects only non-testnet accounts.

Reply:
Changed to use a selector for mainnet coins only.